app/cdap/text/text-en.yaml

```diff
@@ -3100,6 +3100,50 @@ features:
       symbolName: Symbol name
   ServiceEnableUtility:
     serviceNotFound: Cannot find {artifactName} artifact
+  SourceControlManagement:
+    configModal:
+      auth:
+        label: Authentication type
+        pat:
+          label: Personal access token (PAT)
+          token: Token
+          tokenHelperText: If you can’t find the secret containing git token, open a tutorial to learn how to create a new one. 
+          tokenName: Token name
+          tokenNameHelperText: Name your token
+          username: User name
+          usernameHelperText: (Optional) Add a username to your token
+      authHeader: Authentication
+      basicHeader: Basic
+      branch:
+        label: Default branch
+        helperText: (Optional) If not set, it will use the default branch from the github repository
+      cancel: Cancel
+      linkConfirm: Save & Close
+      pathPrefix:
+        label: Path prefix
+        helperText: (Optional) Add Namespace description here
+      provider: 
+        label: Provider
+        helperText: Select Source Control Management tool
+      repoUrl:
+        label: Repository URL
+        helperText: Git repo URL of the form git@<host>/repo.git
+      title: Source Control Management
+      validate:
+        button: validate
+        errorMessage:  
+          0: No errors found       
+          1: "{context} required field is missing"
+          _: "{context} required fields are missing"
+    delete: Delete
```

Comment:
A lot of these words are in common... i'm not sure how i feel about having duplicates. This way of doing internationalization isn't my favorite... hmm i guess this is fine for now